katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -0,0 +1,505 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_katello
+%global confdir hammer
+
+Summary: Katello command plugin for the Hammer CLI
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.11.0
+Release: 1%{?dist}
+Group:   Development/Languages
+License: GPLv3
+URL:     https://github.com/theforeman/hammer-cli-katello
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 0.0.17-3
```

Comment:
Was there a downgrade when moving into SCL?

Reply:
I don't entirely follow. But this was standard practice across all the hammer packages when we SCL'd them.

Reply:
I mean that this is version 0.0.17-3 where ```%version``` is 0.11.0.

Reply:
Right, current version is much higher than this obsoletes as expected unless I am missing something?

Reply:
Exactly, that surprises me.

Reply:
So we cleared this up on IRC: I misread the 0.0.17 as 0.17.0 and was confused.